tools/ratbag-command.c

```diff
@@ -441,8 +441,10 @@ ratbag_cmd_info(const struct ratbag_cmd *cmd,
 		printf("  Profile %d (%s)%s\n", i,
 		       ratbag_profile_is_enabled(profile) ? "enabled" : "disabled",
 		       ratbag_profile_is_active(profile) ? " (active)" : "");
-		if (!ratbag_profile_is_enabled(profile))
+		if (!ratbag_profile_is_enabled(profile)) {
+			ratbag_profile_unref(profile);
 			continue;
```

Comment:
yes, but moving the profile to inside the for loop and making it a `_cleanup_` seems like it's the better way to fix this

Reply:
I somehow missed that comment. Added an extra commit for that on top of this PR